neurodiffeq/generators.py

```diff
@@ -18,6 +18,21 @@ def _chebyshev_second(a, b, n):
     nodes.requires_grad_(True)
     return nodes
 
+def _chebyshev_second_noisy(a, b, n):
+    nodes = torch.cos((torch.arange(n) + (torch.randint(-1,2,(n,))*torch.rand(n)*(b-a)))/ float(n - 1) * np.pi)
```

Comment:
Question about the term `torch.randint(-1,2,(n,))*torch.rand(n)*(b-a))` here:
1. `randint(-1,2,(n,))` will (possibly) contain -1, 0, 1 here. I understand the -1 and 1. But is it supposed to return 0, i.e., keeping something unchanged at all?
2. This is before passing to torch.cos, so that (b-a) factor shouldn't be used.
3. Why don't we changed the whole term to `(torch.rand(n) * 2 - 1) * scale` where scale is some factor? Namely, sampling from a uniform distrution over the interval [-scale, scale]. Notice that the `scale` hasn't taken into account the `/float(n-1) * np.pi` factor at the end.
4. As a reminder, try to be consistent with the styling, including spacing. I would recommend putting a space after every comma `,` (unless followed by another punctuation like `]` or `)`), and putting spaces around binary operators `+`, `-`, `/`, `*`, etc. A special case is `=`, where if used in a function call (keyword argument), you don't put spaces.

Reply:
That's true. I was experimenting with a lot of things that's why it is like that. We can replace it by ` (torch.rand(n) * 2 - 1)`.
Added the changes you mentioned.